/* -*- Mode:protobuf; indent-tabs-mode:nil; -*- */
// More information: https://developers.google.com/protocol-buffers/docs/cpptutorial

syntax = "proto3";

package ndn.ndvr.proto;

// Uma entidade pode ter vários next hops
// Cada Next hop terá seu custo assiciado

message DvInfo {
  message NextHop {
    repeated string router_id = 1;
  }

  message Entry {
    string prefix = 1;
    uint64 seq = 2;
    string originator = 3;
    NextHop next_hops = 4;
    uint32 cost = 5;
  }

  repeated Entry entry = 1;
}
